openstack/swift/templates/health-exporter-deployment.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if $.Values.health_exporter }}
+{{- if not $.Values.health_statsd }}
+kind: Deployment
+apiVersion: extensions/v1beta1
+
+metadata:
+  name: swift-health-exporter
+  labels:
+    release: "{{$.Release.Name}}"
+spec:
+  revisionHistoryLimit: 5
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 2
+  selector:
+    matchLabels:
+      component: swift-health-exporter
+  template:
+    metadata:
+      labels:
+        component: swift-health-exporter
+      annotations:
+        checksum/swift-cluster.etc: {{ include "swift/templates/etc/cluster-configmap.yaml" $ | sha256sum }}
```

Comment:
This is per swift cluster. we do not need. Just add the ring checksum as well `{{- include "swift_conf_annotations" $ | indent 8 }}`